src/components/Validators/ValidatorCard.tsx

```diff
@@ -1,40 +1,45 @@
-import { Card, CardBody, Flex, HStack, Text } from '@chakra-ui/react'
-import { ensure0x, IChainValidator } from '@vocdoni/sdk'
+import { Card, CardBody, Flex, HStack, Link, Text } from '@chakra-ui/react'
+import { ensure0x } from '@vocdoni/sdk'
 import { Trans } from 'react-i18next'
 import { ReducedTextAndCopy } from '~components/Layout/CopyButton'
+import { generatePath, Link as RouterLink } from 'react-router-dom'
+import { RoutePath } from '~constants'
+import { ValidatorFixedType } from '~components/Validators/Detail'
 
-export const ValidatorCard = (validator: IChainValidator) => {
+export const ValidatorCard = (validator: ValidatorFixedType) => {
   return (
     <Card>
-      <CardBody>
-        <Flex gap={2} direction={'column'}>
-          <Text fontWeight='bold' wordBreak='break-all'>
-            {ensure0x(validator.address)}
-          </Text>
-          <Flex fontSize={'sm'} direction={'column'} gap={2}>
-            <HStack gap={1}>
+      <Link as={RouterLink} to={generatePath(RoutePath.Validator, { address: validator.validatorAddress })}>
+        <CardBody>
+          <Flex gap={2} direction={'column'}>
+            <Text fontWeight='bold' wordBreak='break-all'>
+              {validator.name && `(${validator.name})`} {ensure0x(validator.validatorAddress)}
```

Comment:
Create a component to better handle this, and show it like:

With unknown name: `just the sha`
With known name: `Name (sha)`

Idealy, the `(sha)` should be shown in a different style (smaller size and lighter color maybe [?])

Reply:
efcf103

This commit makes an extrapolation of the component, this is the result

![image](https://github.com/user-attachments/assets/607fc2e6-d8c1-4d80-9eb1-e77fea4915f3)

![image](https://github.com/user-attachments/assets/efc85fb8-564b-42c6-9828-b71b8b271603)

![image](https://github.com/user-attachments/assets/3f971b28-caf1-4b71-9fb4-25d76a118468)

![image](https://github.com/user-attachments/assets/c7a52216-521c-454b-b924-cd7cf348a626)